History/HistoryDiff.cpp

```diff
@@ -52,6 +52,8 @@ HistoryDiff::HistoryDiff()
     mToolbar = new QToolBar;
 
     mDiffTo = new QComboBox;
+    mDiffTo->addItem( trUtf8( "All parents" ), DTT_AllParents );
```

Comment:
This looks like a good workaround for the missing "make this setting persistent somewhere" bug :nail_care: